Explorer/Assets/DCL/PluginSystem/World/PersistentEntities.cs

```diff
@@ -5,13 +5,17 @@ namespace DCL.PluginSystem.World
     /// <summary>
     ///     Entities that are created in a world factory and never destroyed while the ECS World is alive
     /// </summary>
-    public readonly struct PersistentEntities
+    public struct PersistentEntities
     {
-        public readonly Entity SceneRoot;
+        public Entity Player { get; private set; }
+        public Entity Camera { get; private set; }
```

Comment:
The same for camera. Do we need `CacheCamera` then?

Reply:
Same as the Player 😆